src/main/java/duke/Instruction.java

```diff
@@ -0,0 +1,8 @@
+package duke;
+
+/**
+ * Extracts a list of instructions for reference.
+ */
```

Comment:
Perhaps you could try adding a more elaborate description of each instruction for future reference?

Reply:
Thanks for the advice! Noted.